### PR TITLE
chore(dev-deps): upgrade to svelte-jester 2.0

### PR DESCRIPTION
**Pull request by benmccann**
`svelte-jester` 2 is a big improvement because it no longer needs to start new node processes thanks to async transformers being added in Jest 27

### CODE REVIEW

**Comment by nickmccurdy** [comment]
Hi, sorry for the delay. The `svelte-jester` update sounds good to me though I'm not experienced with Svelte.

I ran the tests locally and they failed with the same issues that I fixed in #161 (Jest 27 no longer loads JSDOM by default). I added those fixes to this branch and the tests are now passing. I also cleaned up the test code, I don't think DOM cleanup is necessary since Jest creates new environments between test runs anyway.

**Comment by nickmccurdy** [comment]
I got CI working here, but unfortunately it seems like `svelte-jester` 2 requires Node 14, and Node 12 is still officially supported until April. This package still supports all the way back to Node 8 though. How should we handle this?

**Comment by benmccann** [comment]
Ok, I sent a PR to fix that: https://github.com/mihar-22/svelte-jester/pull/83

File: package.json [hunk 1]
```diff
@@ -74,12 +74,12 @@
...
     "eslint-plugin-svelte3": "^3.0.0",
     "husky": "^4.0.9",
-    "jest": "^26.0.1",
+    "jest": "^27.0.0",
     "lint-staged": "^10.0.0",
     "npm-run-all": "^4.1.5",
     "prettier": "^2.0.1",
     "svelte": "^3.0.0",
-    "svelte-jester": "^1.0.3"
+    "svelte-jester": "^2.0.1"
```
Comment (benmccann) [inline package.json:82]:
```suggestion
    "svelte-jester": "^2.1.4"
```

**Comment by benmccann** [comment]
@nickmccurdy I've updated this PR to `svelte-jester` 2.1.4, which supports older versions of Node

**Comment by nickmccurdy** [comment]
Thanks for the fix! Looks great, we seem to have cut test times in half.

**Review by nickmccurdy — APPROVED** [review]

**Comment by nickmccurdy** [comment]
@all-contributors add @benmccann for tests

**Comment by nickmccurdy** [comment]
@all-contributors add @mihar-22 for review

**Pull request merged**